import "mapreduce/yt/interface/protos/extension.proto";

message TFilteredMessage {
    optional string Status = 1 [ (NYT.column_name) = "status" ];
}

message TStatus {
    optional string Status = 1 [ (NYT.column_name) = "status" ];
    optional uint64 Count = 2 [ (NYT.column_name) = "count" ];
}

message TTransposedStatus {
    optional string Date = 1;
    optional uint64 BadPartial = 2;
    optional uint64 GoodPartial = 3;
    optional uint64 Partial206  = 4;
    optional uint64 Total = 5;
}
